stroeer/core/v1/article.proto

```diff
@@ -67,8 +68,9 @@ message Article {
   Metadata metadata = 7;
   repeated Element elements = 8;
   repeated Keyword keywords = 9;
-  repeated int64 onwards = 10;
+  repeated int64 onwards = 10 [deprecated = true];
   map<string, Article> variants = 11;
+  repeated RelatedArticleReference related_article_refs = 13;
```

Comment:
Just naming, it's about IDs. The message itself has the `id` field: 

`repeated RelatedArticleId related_article_ids = 13`

Reply:
Wir haben bereits `stroeer.page.article.v1.RelatedArticle.` welche aber schon aufgelöste Artikel sind, diese hier sind nur die IDs, was man als Referenz bezeichnen könnte.